crates/solidity/outputs/cargo/crate/Cargo.toml

```diff
@@ -25,16 +25,27 @@ categories = [
   "text-editors",
 ]
 
+[[bin]]
+name = "slang_solidity"
+path = "src/main.rs"
+required-features = ["cli"]
+
+[features]
+default = ["cli"]
+cli = ["anyhow", "clap", "serde_json"]
+
 [dev-dependencies]
 solidity_cargo_build = { workspace = true }
 
 [dependencies]
-anyhow = { workspace = true }
 ariadne = { workspace = true }
-clap = { workspace = true }
 semver = { workspace = true }
 serde = { workspace = true }
-serde_json = { workspace = true }
 strum = { workspace = true }
 strum_macros = { workspace = true }
 thiserror = { workspace = true }
+
+# CLI
```

Comment:
I suggest keeping them all alphabetically sorted together. The `cli` distinction should be made through the `[features]` table only.